wallet-new/src/Cardano/Wallet/API/Request/Sort.hs

```diff
@@ -74,6 +74,7 @@ flattenSortOperations (SortOp f fs) = show f : flattenSortOperations fs
 -- much sense, so we restrict ourselves.
 type family SortParams (syms :: [Symbol]) (r :: *) :: [*] where
     SortParams '["balance"] Wallet = '[Core.Coin]
+    SortParams '["timestamp"] Transaction = '[Core.Timestamp]
```

Comment:
@Martoon-00 Shouldn't we change the sorting key to be `created_at` to be consistent with the filtering? 😉

Reply:
tfw stringly kinded programming

Reply:
> tfw stringly kinded programming

Lift your semantic errors at the kind level with this _one weird trick_ -- Haskell consultants hate it!

Reply:
Thanks, missed it :+1:

Reply:
I extracted those `balance`, `id` etc names to type aliases, hope it makes sense.